src/wallet/wallet.cpp

```diff
@@ -2587,6 +2589,17 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
     // all.
     txNew.nLockTime = std::max(0, chainActive.Height() - 10);
 
+    // Activates after Overwinter network upgrade
+    // Set nExpiryHeight to 20 blocks (default) past current blockheight
+    if (NetworkUpgradeActive(chainActive.Height(), Params().GetConsensus(), Consensus::UPGRADE_OVERWINTER)) {
+        txNew.nVersion = 3;
```

Comment:
Unnecessary given `CreateNewContextualCMutableTransaction` is used above.

Reply:
Use CreateNewContextualCMutableTransaction() to get a mutable tx, and then set the expiry height.